### PR TITLE
Added command to debug/dump a DCA

### PR DESCRIPTION
**Pull request by aschempp**
A simple command that dumps a DCA configuration. I needed this because it also shows which callbacks are registered from a service definition.

### CODE REVIEW

**Comment by m-vo** [comment]
I really like this!

File: core-bundle/src/Command/DebugDcaCommand.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    public function __construct(ContaoFramework $framework, Connection $connection)
+    {
+        $this->framework = $framework;
+        $this->connection = $connection;
```
Comment (leofeyer) [inline core-bundle/src/Command/DebugDcaCommand.php]: It seems that `$this->connection` is never used?

**Review by leofeyer — CHANGES_REQUESTED** [review]

File: core-bundle/src/Command/DebugDcaCommand.php (hunk 1 above)
Comment (aschempp) [reply core-bundle/src/Command/DebugDcaCommand.php]: You're right. Dropped in 74b0353053ae2de34e955f16df06f45e6841d4ca

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @aschempp.

**Comment by leofeyer** [comment]
I just realized that the dump does not include the labels from the language array. Is this something we should add?

**Comment by aschempp** [comment]
🤷‍♂ we could, but it's not really relevant in my opinion. If we add it, we also need an option for the language.

**Comment by leofeyer** [comment]
Let's wait and see if someone needs it.

**Pull request merged**